common/src/main/kotlin/flank/common/OutputReportKeys.kt

```diff
@@ -4,4 +4,5 @@ const val OUTPUT_ARGS = "args"
 const val OUTPUT_WEBLINKS = "weblinks"
 const val OUTPUT_TEST_RESULTS = "test_results"
 const val OUTPUT_COST = "cost"
+const val OUTPUT_BILLABLE_MINUTES = "billable-minutes"
```

Comment:
It seems a little unintuitive to me to have keys within the same structure mixing hyphens and underscores. Do you have a strong preference for using a key with a hyphen rather than underscore?

Reply:
Agreed, mixing both is not ideal. Actually, it's already mixed in the code like in https://github.com/Flank/flank/blob/6c6eb5377e6b748a39203719809f4ade0660f11a/integration_tests/src/test/kotlin/utils/OutputReportParser.kt#L20-L30

 I have no strong preference too, but I can change it to use hypen.